### DISCUSSION TITLE
For Sladescar

### DISCUSSION THREAD

**Discussion by joecooldoo** [Hyposyn/Hyposyn#10] [General]
@sladescarofficial I thought I was a "cofounder"???
You removed it here: https://github.com/Hyposyn/Hyposyn/commit/c5152dd2712295341f50d2508197d49d9588895a
And @EarthRulerr still knows tailwind CSS...
And stop copying + pasting code. You're going to break something! 😬

**Comment by prisems** [comment]
I just know TWCSS & CSS now 💀 They call me the CSS wizard. I also work with furret.css

see my updated skills: earthrulerr.is-a.dev

**Comment by sladescarofficial** [comment]
> I thought I was a "cofounder"???

Oh yeahhhhhhhh, I was working on something and i was forgot to add that back, you are. lol.
